_data/pricing.yml

```diff
@@ -1,26 +1,44 @@
-tier1:
-  title: Up to 5 users
-  users_max: 5
-  price: [500, 795]
-  checkout:
-    - price: 500
-      url: /checkout/?plan=1
-    - price: 795
-      url: /checkout/?plan=1&addon=dedicated-support
-  details: |
-    monthly flat fee
+subscription_only:
+  tier1:
+    title: Up to 5 users
+    users_max: 5
+    price: 795
+    checkout:
+      - price: 795
+        url: /checkout/?plan=1
+    details: |
+      monthly flat fee
 
-tier2:
-  title: Users 6 - 50
-  users_max: 50
-  price: [79, 99]
-  checkout:
+  tier2:
+    title: Users 6 - 50
+    users_max: 50
+    price: 79
+    checkout:
+        - price: 79
+          url: /checkout/?plan=2
+    details: |
+      per <span class="help-text light" title="A user is any human or machine that makes use of any of Gruntwork's services (Infrastructure as Code Library, DevOps Training Library, Dedicated Support)" data-toggle="tooltip">user</span> per month
+
+dedicated_support:
+  tier1:
+    title: Up to 10 users
+    users_max: 5
+    price: 2745
+    checkout:
+      - price: "2,745"
+        url: /checkout/?plan=1&addon=dedicated-support
+    details: |
+      monthly flat fee
+
+  tier2:
+    title: Users 11 - 50
+    users_max: 50
+    price: 79
```

Comment:
Not 99?

Reply:
Again, I just felt it was getting too expensive. At this point, we're at 10 users for $2,745/month. I really want our pricing to seem like a no-brainer value and I like the idea that for less than $1,000/year, your dev can access the entire subscription and support.

Reply:
Where is the $1,000/year number from?

Reply:
Oh, I see, the $79 * 12 vs $99 * 12.